ngraph/core/include/openvino/core/preprocess/pre_post_process.hpp

```diff
@@ -5,13 +5,12 @@
 #pragma once
 
 #include "openvino/core/core_visibility.hpp"
+#include "openvino/core/node.hpp"
```

Comment:
still changes are present in the cpp api, please remove them

Reply:
Fixed.